fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractPLSearchTest.java

```diff
@@ -72,7 +72,7 @@ public void createResources() throws Exception {
     @AfterClass
     public void removeSavedResourcesAndResetTenant() throws Exception {
         if (savedResource != null && persistence.isDeleteSupported()) {
-            persistence.delete(getDefaultPersistenceContext(), Basic.class, savedResource.getId());
+//            persistence.delete(getDefaultPersistenceContext(), Basic.class, savedResource.getId());
```

Comment:
do you want to actually do this? 
```suggestion
            persistence.delete(getDefaultPersistenceContext(), Basic.class, savedResource.getId());
```